ipatests/pytest_ipa/integration/tasks.py

```diff
@@ -348,17 +351,20 @@ def install_master(host, setup_dns=True, setup_kra=False, setup_adtrust=False,
             '--forwarder', host.config.dns_forwarder,
             '--auto-reverse'
         ])
+        fw_services.append("dns")
     if setup_kra:
         args.append('--setup-kra')
     if setup_adtrust:
         args.append('--setup-adtrust')
+        fw_services.append("freeipa-trust")
     if external_ca:
         args.append('--external-ca')
 
     args.extend(extra_args)
     result = host.run_command(args, raiseonerr=raiseonerr,
                               stdin_text=stdin_text)
     if result.returncode == 0 and not external_ca:
+        fw.enable_services(fw_services)
```

Comment:
``fw.enable_services(fw_services)`` must be executed in external_ca case, too.

Reply:
Fixed